spec/controllers/ahoy/events_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe Ahoy::EventsController, :type => :controller do
+  routes { Ahoy::Engine.routes }
+
+  it "creates event with other attr" do
+    time = (Time.now - 30).utc.change(:usec => 0)
+    post :create, :id => "11111111-2222-3333-4444-555555555555",
+      :name => "$submit",
```

Comment:
if this were in non-spec code, Rubocop would be complaining about the usage of old hash syntax. not a biggie but an FYI / if you wanna change